packages/blocks/src/api/factory.js

```diff
@@ -490,8 +491,7 @@ export function switchToBlockType( blocks, name ) {
 			transformationsTo,
 			( t ) =>
 				t.type === 'block' &&
-				( isWildcardBlockTransform( t ) ||
```

Comment:
Wildcard `to` transforms (a.k.a. unwrap transforms) don't make sense to be considered when looking for a specific block transform.

E.g. if the quote block registers an unwrap transform, it shouldn't be considered when you transform a quote to a particular other block like pull quote.

The wildcard transforms only make sense from "from" transforms, which means transforming from random mix of blocks to the block that registers it.

E.g. when a quote block registers a "from" wildcard transform, it **should** be considered when switching block type to quote.

Reply:
Should we add this to the transforms docs?